packages/codelists/src/components/codelist-detail/edit-container.js

```diff
@@ -1,18 +1,39 @@
-import React, { useState, useEffect, useCallback } from 'react';
+import React, { useState, useEffect, useCallback, useContext } from 'react';
 import { useParams } from 'react-router-dom';
 import { useSelector } from 'react-redux';
 import { Loading, goBack, goBackOrReplace } from '@inseefr/wilco';
 import { Stores } from 'bauhaus-utilities';
 import { API } from '../../apis';
+import { recalculatePositions } from '../../utils';
+import { TreeContext } from '../tree/treeContext';
 import D from '../../i18n/build-dictionary';
 import { CodeListDetailEdit } from './edit';
 
+const superiorToParent = (child, parent) => {
```

Comment:
These two following methods are duplicated in another JS file. You can create an dedicated file inside the codelist module. Something like utils.js ?